src/tool_getparam.c

```diff
@@ -42,6 +42,14 @@
 
 #include "memdebug.h" /* keep this as LAST include */
 
+#if (SIZEOF_CURL_OFF_T == 4)
+#  define CURL_OFF_T_MAX CURL_OFF_T_C(0x7FFFFFFF)
+#else
+   /* assume CURL_SIZEOF_CURL_OFF_T == 8 */
+#  define CURL_OFF_T_MAX CURL_OFF_T_C(0x7FFFFFFFFFFFFFFF)
+#endif
+#define CURL_OFF_T_MIN (-CURL_OFF_T_MAX - CURL_OFF_T_C(1))
+
```

Comment:
i think rather than dupe these it would be better to have them in one location. iirc they are needed by both src and lib. if strtoofft.h isn't working for you (was included in src by way of curlx i think) how about curl-setup https://github.com/curl/curl/blob/curl-7_57_0/lib/curl_setup.h#L145

Reply:
Agreed!